service/src/java/org/apache/hive/service/cli/thrift/ThriftHttpServlet.java

```diff
@@ -93,6 +93,7 @@
   private final HiveAuthFactory hiveAuthFactory;
   private static final String HIVE_DELEGATION_TOKEN_HEADER =  "X-Hive-Delegation-Token";
   private static final String X_FORWARDED_FOR = "X-Forwarded-For";
+  private static final String X_SAP_HIVE_PASSWDAUTH = "X-SAP-Hive-PasswdAuth";
```

Comment:
this should be x-http-authtype, see also https://github.com/VertiCloud/alti_apiproxy_app-cookbook/blob/master/templates/default/init.lua.erb#L9-L10

The client will send a `x-http-authtype: Basic` header to indicate to the server that Basic auth is requested